### ISSUE TITLE
Set current locale on refresh

### ISSUE THREAD

**Issue by roschaefer** [roschaefer/rundfunk-mitbestimmen#758]
* **I'm submitting a ...**
  * bug report

        
* **What is the current behavior?**
If you login, your `locale` is set. But if you refresh the page while logged in, your current `locale` is set back to your browser locale.
* **What is the expected behavior?**
If I have a `locale` in my user profile, I want the page to be translated on every browser refresh.

* **What is the motivation / use case for changing the behavior?**
Better user experience.

* **If the current behavior is a bug, please tell us (otherwise delete):**
  * Steps to reproduce
        1. Log in
        2. Change language
        3. Refresh

**Comment by roschaefer** [comment]
Feel free to store the current locale in `session.data` in the frontend:

Save
```js
this.get('session').set('data.locale', 'de');
```

On refresh
```js
let lastLocale = this.get('session.data.locale');
if (lastLocale)  // do magic
```

Check out the docs of `ember-intl`:
https://github.com/ember-intl/ember-intl

**Comment by Kachulio1** [comment]
I'll work on this.

**Comment by roschaefer** [comment]
closed via #834
